docs/manual.md

```diff
@@ -476,3 +476,18 @@ following transactions:
 Support for more complex inline math would require substantial changes
 to the parser.
 
+
+Bugs and Contributions
+----------------------
+
+If you find any bugs in ledger2beancount or believe the conversion from
+ledger to beancount could be improved, please [open an
+issue](https://github.com/zacchiro/ledger2beancount/issues).  Please
+include a small test case so we can reproduce the problem.
+
+If you'd like to contribute to ledger2beancount, please submit a [pull
+request on GitHub](https://github.com/zacchiro/ledger2beancount/pulls).
+Please move sure you added a test case under `tests` and updated the
+documentation (`docs/manual.md`, and possibly `README.md` and
+`examples/illustrated.md`).
```

Comment:
Please add here something like "or send a `git format-patch` patch (series) to the authors via email (see `AUTHORS` file)". I'd rather not force people to use GitHub if they don't want to, and I really don't mind patches via email anyway.